frontend/src/metabase/query_builder/components/VisualizationError.jsx

```diff
@@ -22,6 +23,49 @@ const EmailAdmin = () => {
   );
 };
 
+function adjustPositions(error, origSql) {
+  /* Positions in error messages are borked coming in for Postgres errors.
+   * Previously, you would see "blahblahblah bombed out, Position: 119" in a 10-character invalid query.
+   * This is because MB shoves in 'remarks' into the original query and we get the exception from the query with remarks.
+   * This function adjusts the value of the positions in the exception message to account for this.
+   * This is done in mildly scary kludge here in frontend after everything,
+   * because the alternative of doing it in backend
+   * is an absolutely terrifying kludge involving messing with exceptions.
+   */
+  let adjustmentLength = 0;
+
+  // redshift remarks use c-style multiline comments...
+  const multiLineBeginPos = origSql.search("/\\*");
+  const multiLineEndPos = origSql.search("\\*/");
+  // if multiLineBeginPos is 0 then we know it's a redshift remark
+  if (multiLineBeginPos === 0 && multiLineEndPos !== -1) {
+    adjustmentLength += multiLineEndPos + 2; // 2 for */ in itself
```

Comment:
Should we maybe define a constant for `2` so it's more readable?

Reply:
2 is for `*/`, but I can't put `*/` in the var name to make it clearer. I could pull one of those stupid `Paamayim Nekudotayim` things but that wouldn't help. I will try with `end-comment-num-chars`, even though that's confusing in and of itself a bit